api/v1/verticadb_webhook.go

```diff
@@ -1021,6 +1025,108 @@ func (v *VerticaDB) hasValidProbeOverride(allErrs field.ErrorList, fieldPath *fi
 	return allErrs
 }
 
+// validateSandboxes validates if provided sandboxes info is correct
+func (v *VerticaDB) validateSandboxes(allErrs field.ErrorList) field.ErrorList {
```

Comment:
I think you should some checks in this function in their own functions

Reply:
Do you mean each check should have its own function?

Reply:
I'd like to save some spaces for validateVerticaDBSpec(). It will become too long if we have multiple check methods for one component.  And putting sandbox checks in one function will be faster since we looped over all sandboxes/subclusters only once for multiple checks.